tests/functional/ext/code_style/code_style_consider_using_assignment_expr.txt

```diff
@@ -0,0 +1,18 @@
+consider-using-assignment-expr:4:3::"Use 'if (a1 := 2):' instead":HIGH
```

Comment:
The message is nice, I like when the expected code is directly given in the message, very beginner friendly.

Reply:
I agree. Just a bit unfortunate that black will remove the brackets in this example since they are technically redundant -> `if a1 := 2:`. Anyway, I still think it's the right choice to include them in the suggestion as it's a lot safer, especially for beginners.